wGetStarted/L-Dev-SimpleDataSave.md

```diff
@@ -440,7 +466,7 @@ The SQL auto generated template by Interject is formatted based on the ReportSav
 
 There are 2 parameters in the SP. The first one "Interject_RequestContext" was the System Parameter you set up in the Data Portal. Again, this XML contextual information will be passed by Interject to the SP upon execution.
 
-The second parameter is a "TestMode" bit. If you are testing, you can set the value to 1 to print out detailed information upon execution. This will also rollback any changes so your database remains unchanged.
+The second parameter is a "TestMode" bit. If you want to test the SP, set this parameter to 1 and it will print out detailed information upon execution that you can use to diagnosis. In addition, setting the TestMode bit will also keep your database unchanged as it will rollback any changes.
```

Comment:
maybe change, `...set this parameter to 1 and it will print out detailed information upon execution that you can use to diagnosis` to `...set this parameter to 1 and it will print out detailed information upon execution useful for troubleshooting`

Reply:
Changed!